corehq/apps/accounting/templates/accounting/partials/stripe_card_ko_template.html

```diff
@@ -14,23 +16,32 @@
               data-bind="checked: selectedCardType"
               value="saved"
             />
-            {% trans 'Use a saved card' %}
+            {% trans "Use a saved card" %}
           </label>
         </div>
-        <div data-bind="visible: isSavedCard" class="row">
+        <div
+          data-bind="visible: isSavedCard"
+          class="row"
+        >
           <div class="col-sm-8">
             <select
               class="form-control m-0"
-              data-bind="options: savedCards,
-                               optionsText: 'cardName',
-                               value: selectedSavedCard"
+              data-bind="
+                options: savedCards,
+                optionsText: 'cardName',
+                value: selectedSavedCard
+              "
             ></select>
           </div>
           <div class="col-sm-3">
             <button
               type="button"
               class="btn btn-danger"
-              data-bind="event: {click: confirmRemoveSavedCard}"
+              data-bind="
+                event: {
+                  click: confirmRemoveSavedCard
+                }
```

Comment:
It's nitpicky, but I thought we still wanted 4-space indentation for inline javascript?